x/paloma/keeper/keeper.go

```diff
@@ -62,27 +63,32 @@ func NewKeeper(
 	}
 }
 
-func (k Keeper) Logger(ctx sdk.Context) log.Logger {
-	return ctx.Logger().With("module", fmt.Sprintf("x/%s", types.ModuleName))
+func (k Keeper) Logger(ctx context.Context) cosmoslog.Logger {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	return liblog.FromSDKLogger(sdkCtx.Logger()).With("module", fmt.Sprintf("x/%s", types.ModuleName))
 }
 
-func (k Keeper) JailValidatorsWithMissingExternalChainInfos(ctx sdk.Context) error {
-	k.Logger(ctx).Info("start jailing validators with invalid external chain infos")
+func (k Keeper) JailValidatorsWithMissingExternalChainInfos(ctx context.Context) error {
+	liblog.FromSDKLogger(k.Logger(ctx)).Info("start jailing validators with invalid external chain infos")
 	vals := k.Valset.GetUnjailedValidators(ctx)
-
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
 	// making a map of chain types and their external chains
 	type mapkey [2]string
 	mmap := make(map[mapkey]struct{})
 	for _, supported := range k.ExternalChains {
 		chainType := supported.XChainType()
-		for _, cri := range supported.XChainReferenceIDs(ctx) {
+		for _, cri := range supported.XChainReferenceIDs(sdkCtx) {
 			mmap[mapkey{string(chainType), string(cri)}] = struct{}{}
 		}
 	}
 
 	var g whoops.Group
 	for _, val := range vals {
-		exts, err := k.Valset.GetValidatorChainInfos(ctx, val.GetOperator())
+		valAddr, err := utilkeeper.ValAddressFromBech32(k.AddressCodec, val.GetOperator())
+		if err != nil {
+			k.Logger(ctx).Error("Error while getting ValAddress", err)
```

Comment:
So we log the error, then what?

Reply:
I applied the same error catch as added [here](https://github.com/palomachain/paloma/blob/2226e95be60c4972dff44afbbbe7b4cb0ec268c8/x/paloma/keeper/keeper.go#L87)